roles/bootstrap-os/tasks/main.yml

```diff
@@ -71,12 +71,28 @@
     - override_system_hostname
     - ansible_os_family in ['Suse', 'Flatcar Container Linux by Kinvolk', 'Container Linux by CoreOS', 'ClearLinux'] or is_fedora_coreos
 
-- name: "Install ceph-commmon package"
+- name: Install ceph-commmon package
   package:
     name:
       - ceph-common
     state: present
-  when: rbd_provisioner_enabled|default(false)
+  when:
+    - rbd_provisioner_enabled|default(false)
+    - is_fedora_coreos is false
+
+- name: Get installed rpm-ostree packages
+  raw: "export http_proxy={{ http_proxy | default('') }};rpm-ostree status | grep LayeredPackages"
```

Comment:
python installed on coreos like system in script https://github.com/kubernetes-sigs/kubespray/blob/master/roles/bootstrap-os/files/bootstrap.sh

mitogen do not support raw module. 
change `raw` to `shell`

Reply:
bootstrap-os role run in separate play with `strategy: linear`
https://github.com/kubernetes-sigs/kubespray/blob/6bc60e021e39b049ec7135bd4cfb4adfce44d1f7/cluster.yml#L25-L31

Reply:
I put it wrong
https://mitogen.networkgenomics.com/ansible_detailed.html#noteworthy-differences
The raw action executes as a regular Mitogen connection, which requires Python on the target, precluding its use for installing Python.